backend/api/migrations/20210426153720_user-font-table.sql

```diff
@@ -0,0 +1,6 @@
+create table user_font (
+    user_id     uuid    not null references "user" (id) on delete cascade,
+    "index"     int2    not null,
```

Comment:
consider adding a `check ("index" >= 0)` here, all integers in postgres are signed